microcosm_logging/timing.py

```diff
@@ -12,5 +12,5 @@ def elapsed_time(target):
     try:
         yield start_time
     finally:
-        elapsed_time = time() - start_time
-        target["elapsed_time"] = elapsed_time
+        elapsed_ms = (time() - start_time) * 1000
+        target["elapsed_time"] = elapsed_ms
```

Comment:
Keeping the same key name here; I can be persuaded to change it / keep around the old value if needed. I would prefer not to have to introduce a new field for just `elapsed_time_ms`.